handle/Timer.lua

```diff
@@ -1,30 +1,190 @@
 require('CeresStdLib.base.Native')
 require('CeresStdLib.base.Log')
-local free_timers = {}
+
+local Timer 		= {}
+local free_timers	= {}
+local MAX_PUSHES	= 10000		--	Maximum amount of times push is called before garbage collection.
+
+free_timers[0]		= 0
```

Comment:
Again, use unique names.

Reply:
replaced [0] with .free, and [1] with .steps